staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -663,7 +663,7 @@ func (r *crdHandler) tearDown(oldInfo *crdInfo) {
 
 	for _, storage := range oldInfo.storages {
 		// destroy only the main storage. Those for the subresources share cacher and etcd clients.
-		storage.CustomResource.DestroyFunc()
+		storage.CustomResource.Store.(*genericregistry.Store).DestroyFunc()
```

Comment:
Could we put DestroyFunc() on the new Store interface?

Reply:
https://github.com/kcp-dev/kubernetes/pull/68/files